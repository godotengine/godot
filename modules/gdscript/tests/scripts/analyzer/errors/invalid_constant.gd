func test():
	var i = 12
	# Constants must be made of a constant, deterministic expression.
	# A constant that depends on a variable's value is not a constant expression.
	const TEST = 13 + i
