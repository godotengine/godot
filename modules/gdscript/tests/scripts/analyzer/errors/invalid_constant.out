GDTEST_ANALYZER_ERROR
>> ERROR at line 5: Assigned value for constant "TEST" isn't a constant expression.
